jdaviz/data/icons/convert.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+inkscape --without-gui --export-png=$PWD/pan_wcs.png --export-width=125 --export-height=125 $PWD/pan_wcs.svg
```

Comment:
What is this for? Is this executed at run time?

Reply:
It's just so that if we add more svg icons we can convert them all in a consistent way. It's not meant to be part of the installed package. Maybe I can move it to the root and call it ``.convert_icons_to_png.sh`` or something

Reply:
It is only one line of command. Can we just move it to https://github.com/spacetelescope/jdaviz/wiki ?

Reply:
True, my point was more that it might grow as more icons are added and we don't want all icons to be added in an inconsistent way. What about putting it as a comment in the ``__init__.py`` of the icons directory?

Reply:
Not sure how discoverable that is. How about a short paragraph in dev docs then?

Reply:
Actually let me just remove it - we really should just get the SVG icons to work.

Reply:
Which I'll look into in a separate PR but my point is we don't need to document how to get the SVG.